solidity_contracts/src/UniswapV2/test/ERC20.sol

```diff
@@ -1,9 +1,13 @@
 pragma solidity =0.5.16;
 
-import '../UniswapV2ERC20.sol';
+import "../UniswapV2ERC20.sol";
 
 contract ERC20 is UniswapV2ERC20 {
     constructor(uint _totalSupply) public {
         _mint(msg.sender, _totalSupply);
     }
+
+    function mint(address to, uint value) external returns (bool) {
```

Comment:
so it returns always false ?

Reply:
I think in OZ's spec it just always returns true

Reply:
ah oop